<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>488</width>
    <height>448</height>
   </rect>
  </property>
  <property name="acceptDrops">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="CentralWidget">
   <property name="acceptDrops">
    <bool>true</bool>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_5">
    <item>
     <widget class="QWidget" name="MainWindow_2" native="true">
      <property name="acceptDrops">
       <bool>true</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">QWidget#MainWindow_2 {
border: 1px solid black;
background-color:#1B262C;
}
</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QFrame" name="controlFrame">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>26</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>32</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QFrame" name="controlSpacing">
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QFrame" name="controlButtons">
            <property name="maximumSize">
             <size>
              <width>60</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">background-repeat: no-repeat;
background-position: center;
</string>
            </property>
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <property name="spacing">
              <number>0</number>
             </property>
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QPushButton" name="minimize_button">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>0</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>26</width>
                 <height>26</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true">background-image: url(:/newPrefix/icons/icons8-minimize-window-24.png);
border-top: 0px solid black;
</string>
               </property>
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="clos_button">
               <property name="maximumSize">
                <size>
                 <width>26</width>
                 <height>26</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true">background-image: url(:/newPrefix/icons/icons8-close-window-24.png);
border-top: 0px solid black;</string>
               </property>
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="Title" native="true">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>80</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">font: 36pt &quot;Cooper Black&quot;;
color:#3282B8;
</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QLabel" name="histLabel">
            <property name="layoutDirection">
             <enum>Qt::RightToLeft</enum>
            </property>
            <property name="text">
             <string>HIRST DRAWING</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="Body" native="true">
         <property name="acceptDrops">
          <bool>true</bool>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QLabel" name="image_viewer">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="acceptDrops">
             <bool>true</bool>
            </property>
            <property name="styleSheet">
             <string notr="true">font: 75 8pt &quot;Georgia&quot;;
border: 4px dashed #0F4C75;
text-align: center;
color:#3282B8;</string>
            </property>
            <property name="text">
             <string>Drag Image Here</string>
            </property>
            <property name="scaledContents">
             <bool>false</bool>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="hist_button_frame">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>40</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
background-color: 3282B8;
border: none;
border-left: 1px solid #0F4C75;
border-right: 1px solid #0F4C75;
border-bottom: 5px solid #0F4C75;
}

QPushButton:hover {
background-color: #151d22;
border-left: 1px solid #0F4C75;
border-right: 1px solid #0F4C75;
border-bottom:  5px solid #0F4C75;
}

QPushButton:pressed {
background-color: #151d22;
border-left: 1px solid #0F4C75;
border-right: 1px solid #0F4C75;
border-top:  5px solid #0F4C75;
}</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>20</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>20</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item alignment="Qt::AlignHCenter">
           <widget class="QRadioButton" name="hirst_radio_button">
            <property name="styleSheet">
             <string notr="true">font: 14pt &quot;Constantia&quot;;
color: #3282B8;</string>
            </property>
            <property name="text">
             <string>Hirst</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item alignment="Qt::AlignHCenter">
           <widget class="QRadioButton" name="color_radio_button">
            <property name="styleSheet">
             <string notr="true">font: 14pt &quot;Constantia&quot;;
color: #3282B8;</string>
            </property>
            <property name="text">
             <string>Color Palette</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="hirst_button">
            <property name="styleSheet">
             <string notr="true">font: 14pt &quot;Constantia&quot;;
color: #3282B8;




</string>
            </property>
            <property name="text">
             <string>Create</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="img.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>minimize_button</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>showMinimized()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>405</x>
     <y>55</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>clos_button</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>515</x>
     <y>55</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
